linodecli/api_request.py

```diff
@@ -339,3 +345,33 @@ def _handle_error(ctx, response):
             columns=["field", "reason"],
         )
     sys.exit(1)
+
+
+def _check_retry(response):
+    """
+    Check for valid retry scenario, returns true if retry is valid
+    """
+    if response.status_code == 408:
+        # request timed out
+        return True
+    if response.status_code == 429:
+        # rate limit exceeded
+        return True
+    if response.headers:
+        if (
+            response.status_code == 400
+            and response.headers.get("Server") == "nginx"
+            and response.headers.get("Content-Type") == "text/html"
+        ):
+            # nginx html response
+            return True
+    return False
+
+
+def _get_retry_after(headers):
+    if headers.get("Retry-After"):
+        retry_str = headers.get("Retry-After")
+        if retry_str == "":
+            return 0
+        return int(retry_str)
+    return 0
```

Comment:
Looks good! Just a small suggestion.

```suggestion
    retry_str = headers.get("Retry-After", "")
    return int(retry_str) if retry_str else 0
```